src/utils/embeds.ts

```diff
@@ -3,7 +3,8 @@ import { vars } from '../config';
 
 const NOTIF_CHANNEL_ID: string = vars.NOTIF_CHANNEL_ID;
 
-export const EMBED_COLOUR = '#0099ff';
+export const DEFAULT_EMBED_COLOUR = '#0099ff';
+export const INFO_EMBED_COLOUR = '#4287f5';
```

Comment:
What do you think about moving all consts (across all files) to files in a `constants` directory or something similar?